### PR TITLE
FP-202 MiniPlayer Allowing trigger MiniPlayer by code

### PR DESCRIPTION
**Pull request by leroybm**

### CODE REVIEW

File: test/html/vod_miniplayer.tpl.html [hunk 1]
```diff
@@ -101,14 +101,34 @@
...
 
+        console.log('%cTEST INSTANCE', 'color: #fff; font-weight: bold; background-color: #BADA55; padding: 3px 6px; border-radius: 3px;', instance);
+
         instance.on('miniPlayerToggle', (event) => console.log(`[Event API] miniPlayerToggle`, event, event.detail.isToggledOn));
 
         Array.from(document.getElementById('setupForm').children).forEach(child => child.remove());
         const refreshPage = document.createElement('button');
         refreshPage.onclick = () => location.reload();
-        refreshPage.innerText = 'Reset';
+        refreshPage.innerText = 'Reset Test';
```
Comment (ZentoDH) [inline test/html/vod_miniplayer.tpl.html:111]: Should this still have "test" in it?

Comment (leroybm) [reply test/html/vod_miniplayer.tpl.html:111]: Yeah, it's the test file, the button is to reload the page to start the test from the start :)

Comment (ZentoDH) [reply test/html/vod_miniplayer.tpl.html:111]: In that case, approved. LGTM

**Review by ZentoDH — APPROVED** [review]

File: test/html/vod_miniplayer.tpl.html [hunk 2]
```diff
@@ -101,14 +101,34 @@
             } : {})
         });
 
+        console.log('%cTEST INSTANCE', 'color: #fff; font-weight: bold; background-color: #BADA55; padding: 3px 6px; border-radius: 3px;', instance);
```
Comment (fpfcarvalho) [inline test/html/vod_miniplayer.tpl.html:104]: this console log should stay here?

**Review by fpfcarvalho — APPROVED** [review]

**Pull request merged**